nuvion_app/runtime/config_guard.py

```
@@ -299,15 +308,28 @@ def update(key: str, new_value: str, reason: str) -> None:
 
 
-def _collect_effective_values(fields: List[Dict[str, str]], file_values: Dict[str, str]) -> tuple[Dict[str, str], Dict[str, Tuple[str, str]]]:
+def _collect_effective_values(
+    fields: List[Dict[str, str]],
+    file_values: Dict[str, str],
+    original_file_values: Dict[str, str] | None = None,
+) -> tuple[Dict[str, str], Dict[str, Tuple[str, str]]]:
     effective = dict(file_values)
     overrides: Dict[str, Tuple[str, str]] = {}
+    original = original_file_values or {}
     for field in fields:
```

Ruff (0.16.2)

[error] Import `field` from line 4 shadowed by loop variable

(F402)